openmetadata-ui/src/main/resources/ui/src/pages/ProfilerDashboardPage/ProfilerDashboardPage.tsx

```diff
@@ -136,8 +136,9 @@ const ProfilerDashboardPage = () => {
   const fetchTableEntity = async () => {
     try {
       const fqn = isColumnView
-        ? getTableFQNFromColumnFQN(decodedEntityFQN)
-        : decodedEntityFQN;
+        ? getTableFQNFromColumnFQN(entityTypeFQN)
+        : entityTypeFQN;
```

Comment:
We have tested both cases? with isColumnView & without isColumnView

Reply:
So earlier we use to have both scenarios but, now the table profile is moved to the table details page -> profiler tab, so now we only have one scenario which is column view, I have attached a video for the same.